lib/registries/addon/modifiers/with-branding.ts

```diff
@@ -0,0 +1,24 @@
+import Modifier from 'ember-oo-modifiers';
+
+import Brand from 'registries/services/brand';
+
+import { inject as service } from '@ember/service';
+
+class WithBrandingModifier extends Modifier {
+    @service brand!: Brand;
+
+    didInsertElement() {
+        const elementStyle = this.element.style;
+        const { currentBrand } = this.brand;
+
+        if (currentBrand) {
+            elementStyle.setProperty('--primary-color', currentBrand.get('primaryColor'));
+            elementStyle.setProperty('--secondary-color', currentBrand.get('secondaryColor'));
+            elementStyle.setProperty('--navbar-logo-img-url', currentBrand.get('navbarLogoImage'));
+            elementStyle.setProperty('--hero-logo-img-url', currentBrand.get('heroLogoImage'));
+            elementStyle.setProperty('--hero-background-img-url', currentBrand.get('heroBackgroundImage'));
```

Comment:
it shouldn't be necessary to use `get` anymore, just plain property access should work -- we're trying to write new code in an octane-y way, as much as is reasonable

Reply:
Hmm I was doing it the octane way before but the ts compiler was complaining and instructing to use `.get`. Let me try again

Reply:
I actually get a runtime error:

```
Uncaught Error: Assertion Failed: You attempted to access the `primaryColor` property (of <(unknown):ember465>).
Since Ember 3.1, this is usually fine as you no longer need to use `.get()`
to access computed properties. However, in this case, the object in question
is a special kind of Ember object (a proxy). Therefore, it is still necessary
to use `.get('primaryColor')` in this case.

If you encountered this error because of third-party code that you don't control,
there is more information at https://github.com/emberjs/ember.js/issues/16148, and
you can help us improve this error message by telling us more about what happened in
this situation.
    at assert (index.js:128)
    at Object.get (core_object.js:202)
    at WithBrandingModifier.didInsertElement (with-branding.js:65)
    at triggerLifecycleHook (modifier-manager.js:19)
    at ModifierManager.installModifier (modifier-manager.js:48)
    at index.js:8285
    at runInAutotrackingTransaction (index.js:927)
    at track (index.js:1193)
    at InteractiveCustomModifierManager.install (index.js:8285)
    at Transaction.commit (runtime.js:3006)
```

Thoughts?

Reply:
ohhh, i hate proxies. i guess instead of getting `model.brand` in `afterModel`, you could get `model.belongsTo('brand').value()`, which should be safely non-null because the brand was just embedded in the model hook...

does that seem reasonable, or am i clinging too hard to "never use `.get`"?

Reply:
That seems reasonable. Let me give that a shot.